bower.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "json-refs",
   "main": "browser/json-refs.js",
-  "version": "1.3.0",
```

Comment:
My worry is that it will break existing tooling like [mversion](https://github.com/mikaelbr/mversion).  I've not verified this but I thought that `mversion` would use the values in `bower.json` and `package.json` when revving things.  Can you check to see if `mversion` still works with the `version` missing in `bower.json`?